render/mapping.go

```diff
@@ -121,7 +121,12 @@ func MapContainerIdentity(m RenderableNode, _ report.Networks) RenderableNodes {
 		rank  = m.Metadata[docker.ImageID]
 	)
 
-	return RenderableNodes{id: NewRenderableNodeWith(id, major, minor, rank, m)}
+	node := NewRenderableNodeWith(id, major, minor, rank, m)
+	if imageID, ok := m.Metadata[docker.ImageID]; ok {
+		hostID, _, _ := report.ParseContainerNodeID(m.ID)
+		node.Origins = node.Origins.Add(report.MakeContainerNodeID(hostID, imageID))
```

Comment:
Hmm. This definition

> I would describe origins as the list of node ids for which you want info to appear in the details panel...

is too loose, but re: host ID, I'll agree that my definition is too strict. How about this: origins are nodes which have a strict parent relationship to the renderable node. Accordingly, how do you feel about renaming the field Parents?